client/src/layouts/default/AppBar.vue

```diff
@@ -211,6 +215,20 @@ export default {
 			}
 		};
 
+		const checkNextLaunch = () => {
+			nextlaunch.value = localStorage.getItem("nextlaunch") == "true";
+			if (!nextlaunch.value) {
+				router.push({ name: "NextLaunch" });
+			}
+		};
+
+		const checkMaintenance = () => {
+			maintenance.value = localStorage.getItem("maintenance") == "false";
+			if (!maintenance.value) {
```

Comment:
It should be if `if (maintenance.value) ` remove `!`

Reply:
> It should be `if (maintenance.value) ` remove `!`

I made it == false checking, that's why I added the '!'
bas realized we commonly use it checking for true, fa I'll revert both